app/models/setting/ansible.rb

```diff
@@ -10,6 +10,7 @@ class << self
       # rubocop:disable BlockLength
       def load_defaults
         return unless super
+        Setting::BLANK_ATTRS.push('ansible_ssh_private_key_file')
```

Comment:
The issue here was that it tried to create the setting as '' by default. The validation only allows this when the setting name is in `BLANK_ATTRS`. Since `BLANK_ATTRS` was only updated after trying to create the setting, instead of **before**, it didn't work.

Reply:
yup, I figured out, reviewed and tested yesterday, you've got my ACK if that helps merging :-)